packages/app/src/layouts/default.vue

```diff
@@ -133,10 +146,13 @@ const showEnableNotificationsBanner = computed(() => {
       Date.now() > new Date(query.data.value?.localSettings.preferences.dismissNotificationBannerUntil).getTime() : true)
 })
 
+const pause = computed(() => !showHeader.value)
+
 const query = useQuery({
   query: MainAppQueryDocument,
-  pause: !showHeader.value,
+  pause,
```

Comment:
This seems to work better if you pass in a `ref`, as opposed to the inner value property.